db/migrate/20180525130651_add_stem_offered_to_institutions.rb

```diff
@@ -0,0 +1,5 @@
+class AddStemOfferedToInstitutions < ActiveRecord::Migration
+  def change
+    add_column :institutions, :stem_offered, :boolean, default: false, index: true
```

Comment:
@saneshark can you break this migration out so we don't lock the table? Use a change default in a separate migration? Not concerned with the index, since that won't lock reads.